docs/API.md

```diff
@@ -166,15 +166,17 @@ query HelloWorld {
 
 <h5 id="graphql-tags">Tags</h5>
 
-| Tag              | Description                                               |
-|------------------|-----------------------------------------------------------|
-| graphql.document | The original GraphQL document.                            |
+| Tag               | Description                                               |
+|-------------------|-----------------------------------------------------------|
+| graphql.document  | The original GraphQL document.                            |
+| graphql.variables | The variables applied to the document.                    |
 
 <h5 id="graphql-config">Configuration Options</h5>
 
-| Option  | Default                                          | Description                            |
-|---------|--------------------------------------------------|----------------------------------------|
-| service | *Service name of the app suffixed with -graphql* | The service name for this integration. |
+| Option          | Default                                          | Description                                                      |
+|-----------------|--------------------------------------------------|------------------------------------------------------------------|
+| service         | *Service name of the app suffixed with -graphql* | The service name for this integration.                           |
+| filterVariables | `variables => variables`                         | A callback that allows filtering of variables before submission. |
```

Comment:
I would use a non-verb name to allow extending this in the future to support for example an array of object paths or a definition object such as blacklists or whitelists.

Reply:
Actually, the default should be to not store the variables at all. The user **has** to provide a filter. This will avoid sensitive data to be sent by default.